app/views/content/life-as-a-teacher/teaching-as-a-career/from-qualified-teacher-to-head-of-biology/_article.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>I knew teaching was for me</h2>
+      <p>From an early age, I knew a teaching career was for me. I could not imagine working a typical nine-to-five desk job. Every day in teaching brings a new challenge and new situations, which is what makes the job so exciting.</p>
+      <p>When your pupils have those ‘light bulb’ moments, hearing the sudden ‘So that’s how it works!’ and seeing them take your subject into A level and beyond makes the job so worthwhile.</p>
+      <h2>My career development</h2>
+      <p>I am extremely proud of my teaching achievements and career progression so far. The encouragement and support from my head of faculty really helped; they saw the potential in me and I’m so grateful for that opportunity.</p>
+      <p>With my leadership skills, my department has gone from strength to strength. We have managed to increase uptake of students from a class of 20 A level students to 50. This was a great achievement, and I could not have done this without the support of my colleagues and the leadership courses I attended.</p>
+      <p>I also act as a teaching and learning mentor, which allows me to work with staff from all departments to enhance and develop their teaching and learning.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I love being able to work with teachers of all backgrounds, experienced to newly qualified, sharing good practice and developing whole-school strategies.",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+      <p>I am currently facilitating an early career teachers (ECT) leadership programme for teachers across the borough, which I am thoroughly enjoying.</p>
```

Comment:
this won't be current now - do we need to reword or leave as is?

Reply:
I think leave it as-is unless we have a more accurate update of what they're doing now?